src/Lean/Widget/InteractiveDiagnostic.lean

```diff
@@ -160,21 +152,19 @@ def msgToInteractiveDiagnostic (text : FileMap) (m : Message) (hasWidgets : Bool
     | none        => low
   let range : Range := ⟨low, high⟩
   let fullRange : Range := ⟨low, fullHigh⟩
-  let severity := match m.severity with
+  let severity? := match m.severity with
```

Comment:
Did the type change? If not, that's really confusing.

Reply:
I just wanted it to have the same name as the field so I could use the abbreviated syntax for structure initialization at the end of the function

Reply:
Apparently it's always been `Option`. An explicit `some` sounds good!